### PR TITLE
Add selectedShape and selectedShapeType to allow for initially selected servers

### PR DESCRIPTION
**Pull request by TwoPlusTwoOne**
When components is initialized, it may receive `selectedShape` and `selectedShapeType` properties from its parent.

Behavior has been made in such a way that the order of `selectedShape`, `selectedShapeType` and `amf` properties doesn't matter (learned the hard way after `selected` and `value` ordering problem in this component).

Notable changes:
- `_amfChanged` now calls `_handleShapeChange` instead of `_updateServers` directly. But the workflow for when there are no `selectedShape` and `selectedShapeType` remains with the same result.
- `_updateServers` will check if the `id` and `type` match. This way, if an `id` and `type` which are not valid are supplied, the component will not attempt to update the servers

### CODE REVIEW

File: src/api-server-selector.js [hunk 1]
```diff
@@ -110,6 +110,11 @@ export class ApiServerSelector extends EventsTargetMixin(AmfHelperMixin(LitEleme
        * Note, this does nothing when custom element is rendered.
        */
       opened: { type: Boolean },
+
+      /**
+       *
+       */
+      selectedValue: { type: String },
```
Comment (jarrodek) [inline src/api-server-selector.js:117]: How's this different from `value`? Where's the doc? :)

Comment (TwoPlusTwoOne) [reply src/api-server-selector.js:117]: I swear I was about to.. hence the comment block. Lol.

File: src/api-server-selector.js [hunk 2]
```diff
@@ -372,6 +377,16 @@ export class ApiServerSelector extends EventsTargetMixin(AmfHelperMixin(LitEleme
     this.selectIfNeeded();
   }
 
+  set selectedValue(value) {
```
Comment (jarrodek) [inline src/api-server-selector.js:380]: Weird, there's no comment I added earlier.
Why this change? What is this? How's this different from the `value`?

**Comment by jarrodek** [comment]
In this case do we still need navigation selection handler? If the parents now have to pass this 2 properties then the navigation event is redundant.
